packages/esm-ward-app/src/ward-view/ward-view.component.tsx

```diff
@@ -80,18 +117,27 @@ const WardViewByLocation = ({ location }: { location: Location }) => {
           ))}
       </>
     );
-  } else {
+  } else if (errorLoadingLocation) {
     return (
       <InlineNotification
         kind="error"
         lowContrast={true}
         title={t('errorLoadingWardLocation', 'Error loading ward location')}
         subtitle={
-          error?.message ??
+          errorLoadingLocation?.message ??
           t('invalidWardLocation', 'Invalid ward location: {{location}}', { location: location.display })
         }
       />
     );
+  } else {
+    return (
+      <InlineNotification
+        kind="error"
+        lowContrast={true}
+        title={t('errorLoadingWardLocation', 'Error loading admitted patients')}
+        subtitle={errorLoadingPatients?.message}
```

Comment:
```suggestion
        subtitle={errorLoadingPatients.message}
```

If `errorLoadingPatients` is falsy at this point then we are in trouble.